enterprise/views.py

```diff
@@ -867,10 +890,21 @@ def post(self, request):
             # EnterpriseCourseEnrollment will be created when the user will select a course run.
             # A CourseEnrollment record will be created and on the post signal of the CourseEnrollment,
             # an EnterpriseCourseEnrollment record will also get created.
-
             if course_id and self.is_course_run_id(course_id):
                 try:
-                    self.create_enterprise_course_enrollment(request, consent_record, course_id)
+                    self.create_enterprise_course_enrollment(request, consent_record, course_id, license_uuid)
+                    if license_uuid:
+                        # Use enrollment api client to create an lms course enrollment
+                        enrollment_api_client = EnrollmentApiClient()
+                        # Ensure the 'verified' track exists for the given course
+                        # If not, enroll in the 'audit' track
+                        course_modes = enrollment_api_client.get_course_modes(course_id)
+                        course_mode = 'verified' if 'verified' in course_modes else 'audit'
```

Comment:
Just checking my understanding here, this means https://github.com/edx/frontend-app-learner-portal-enterprise/pull/58 won't be needed right?

Reply:
correct!